src/cmd/docs.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+import defaultUrlOpener from 'open';
+
+import {createLogger} from '../util/logger';
+
+const log = createLogger(__filename);
+
+type DocsOptions = {
+	openUrl?: typeof defaultUrlOpener,
+}
+
+export default function docs(
+  params: Object, {openUrl = defaultUrlOpener}: DocsOptions = {}
+	) {
+  const url = 'https://developer.mozilla.org/en-US/Add-ons' +
```

Comment:
If you move this to the top as:
````js
export const url = '...';
````
then you could import it in your test like:
````js
import defaultDocsCommand, {url} from '../../../src/cmd/docs';
````
and then you wouldn't have to copy/paste it to make the mock assertion.